variables.tf

```diff
@@ -128,7 +129,8 @@ variable "read_pool_instance" {
     database_flags    = optional(map(string))
     availability_type = optional(string)
     gce_zone          = optional(string)
-    machine_cpu_count = optional(number, 2)
+    machine_cpu_count = optional(number, 2),
+    ssl_mode          = optional(string, "ALLOW_UNENCRYPTED_AND_ENCRYPTED"),
```

Comment:
Change it to:
ssl_mode = optional(string)
require_connectors = optional(string)

Reply:
Fixed. PLTL.